README.md

```diff
@@ -1005,6 +1030,23 @@ The `not` and `case: ignore` directives can both be applied on the same rule. Th
   - [ X-Forwarded-For, { value: a, not: prefix, case: ignore } ]
 ```
 
+#### meta headers
+In addition to the directives that check the actual content of individual
+fields, Proxy Verifier supports meta-fields denoted by an `@` prefix that serve
+as directives for the tool to perform custom operations. The following
+demonstrates the usage of the `@hasfields` meta-field to verify that the
+`header` node has at least one field:
+
+```YAML
+proxy-response:
+  headers:
+    fields:
+    - [ :status, { value: 200, as: equal } ]
+    - [ Cache-Control, private ]
+    - [ Content-Length, { value: '16', as: equal } ]
+    - [ "@hasfields", { value: true, as: equal } ]
+```
```

Comment:
This example seems a little strange since it verifies two fields then verifies that there are fields. The other two would imply the latter.

What is the use case for the `@hasfields` meta header? When would we not be verifying the existence of some specific header but would want to verify that there are some generic headers? Maybe this is more of a question for @keess2 .

Reply:
To be honest this is not clear to me too... Maybe @keess2 can weigh in?

Reply:
Gotcha. Thank you @keess2 for the context.

Let's do this: let's handle meta fields as a separate PR as the need arises. In the meantime, let's move forward with the other work that you and @lzx404243 did without `@hasfields` and get this landed. I've also been working on Expect: 100-Continue changes and HTTP/3 timeout changes which will be good to land in a release as well.

Thanks @lzx404243 and @keess2 .

Reply:
Updated the PR leaving off the `@hasfields` logic.